plugins/zip-extractor/src/index.ts

```diff
@@ -28,8 +28,9 @@ export const ZipExtractor = () => {
             progress: 50,
             info: 'Uploading files',
           })
-          zipEntries.forEach(async (zipEntry) => {
+          const uploadPromises = zipEntries.map(async (zipEntry) => {
             if (
+              !zipEntry.entryName.startsWith('__MACOSX') &&
```

Comment:
❔ `__MACOSX` is a directory so why doesn't `isDirectory` work?

Reply:
@hansjhoffman `isDirectory` matches on `__MACOS` but doesn't on `__MACOS/file.ext`. We want to traversal each directory and upload the contents. This check stops traversal of the `__MACOS` directory since we know the contents are not relevant.